pkgs/top-level/all-packages.nix

```diff
@@ -6410,7 +6410,7 @@ in
 
   pipr = callPackage ../applications/misc/pipr { };
 
-  pipreqs = callPackage ../tools/misc/pipreqs { };
+  pipreqs = python3Packages.callPackage ../tools/misc/pipreqs { };
```

Comment:
Please avoid this pattern, use the `callPackage` of your current scope. That way we guarantee we get a compatible versions of packages.

Reply:
should we remove the examples in documentation then?

Reply:
Pretty sure there is no such example for Python. I know it is there for Qt.

Reply:
I guess I just picked up this habit from other packages.

Should we have a, "Instead of, XXX, do YYY" section in the documentation? I guess we don't "encourage" it, but we don't really discourage the behavior either.

Reply:
done